src/containers/Checkin/CheckinPic/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Flex, ImagePicker } from "antd-mobile"
+import './checkinPic.css'
+
+const CheckinPic = ({ files, onChange }) => {
+  return (
+    <div>
+      <div className='checkin__content--word-title'>
+        打卡图片（可选）
+      </div>
+      <Flex.Item>
+        <div className='checkin__img--checkin-pic'>
+          <ImagePicker
+            files={files}
+            length={2}
+            onChange={onChange}
+            selectable={files.length === 0}
```

Comment:
`selectale ={!files.length}`

Reply:
Actually,  `selectable={files.length === 0}` is exactly what I want. 

If `selectale === true`, which means you can add more pictrues, but I only need one picture for user's avatar.

For more detail, you can check the doc [here](https://mobile.ant.design/components/image-picker-cn/).

Reply:
> `selectale ={!files.length}`

Actually, selectable={files.length === 0} is exactly what I want.

If selectale === true, which means you can add more pictrues, but I only need one picture for user's avatar.

For more detail, you can check the doc here.

Reply:
`!files.length` does the same job as `files.length === 0`

Reply:
> `!files.length` does the same job as `files.length === 0`

My bad. I'll fix it immediately